### DISCUSSION TITLE
python

### DISCUSSION THREAD

**Discussion by 0x7n8** [community/community#64285] [Programming Help]
### Body

hi, im new to python programming, anyone can help me to get path or something else? btw i used python for pentesting, thanks

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by 0x7n6** [comment]
hmm mybe you can explore on tcm course "python 101"

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
